.github/workflows/Docker.yml

```diff
@@ -0,0 +1,59 @@
+name: Docker
+
+on:
+  workflow_run:
+    workflows: ["Docker CI"]  # Run only if "Docker CI" workflow succeeds
+    types:
+      - completed
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: "Release Version (e.g., 1.0.0)"
+        required: false
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package: [ingest, iot-config, iot-packet-verifier, iot-verifier, mobile-config, mobile-packet-verifier, mobile-verifier, reward-index]
```

Comment:
Do we need?

- boost-manager
- poc-entropy
- price

Reply:
I did everything with a "runner" but I don't know if we actually run everything, question for Brian

Reply:
We run those 3 services today, is that the question?

Reply:
Yes